tst/components/ui/Modal/ModalButtonGroup.test.tsx

```diff
@@ -16,7 +16,7 @@ describe('ModalButtonGroup', () => {
   const secondaryButtonLbl = 'test-secondary';
   const onClose = jest.fn();
   const labelID = 'test-label';
-  const testContext = { onClose, labelID, dismissible: true, };
+  const testContext = { onClose, labelID, dismissible: true };
```

Comment:
[Learning] Here linting removed the trailing comma it seems, so can you help understand in which scenario trailing comma is accepted and in which we remove it?

Reply:
We can add the commas rules later, if not auto-fixing it.
But as @ldai1 mentioned, we can add the `comma-dangle` rule if it is easily auto-fixing the trailing commas: https://eslint.org/docs/rules/comma-dangle#options